packages/web-service/src/services/s3-upload.js

```diff
@@ -0,0 +1,40 @@
+import { S3 } from '@aws-sdk/client-s3'
+import fs from 'fs'
+const accountID = '514125100797'
+const region = 'eu-west-2'
+
+export async function s3FileUpload (applicationName, fileName, path) {
+  const s3inst = new S3({
+    region: region,
+    credentials: {
+      secretAccessKey: '0fSeWQAZAi0KeIhydPWEpwZcihCV09nfp5/gx/ly',
```

Comment:
Please remove these values. Is this your own account? If so its compromised so you might want to rotate the keys

Reply:
Don't worry I sorted that out a while ago and just left them in as example values.

Reply:
I think it might trigger some alerts on the repo so it has to come out. Probably best to remove the whole credentials clause, it will be handled in the environment and is unnecessary